psi4/src/psi4/fisapt/fisapt.cc

```diff
@@ -118,8 +118,8 @@ void FISAPT::localize() {
     ranges.push_back(vectors_["eps_focc"]->dimpi()[0]);
     ranges.push_back(vectors_["eps_occ"]->dimpi()[0]);
 
-    std::shared_ptr<fisapt::IBOLocalizer2> local =
```

Comment:
While this is tangential to the purpose of the PR, please do the trivial cleanup of using `auto` in all three code blocks where the variable is defined.